assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -164,13 +164,13 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 					case 'Enter':
 					case 'Space':
 						event.preventDefault();
-						this.changeActiveTab( event.currentTarget.getAttribute( 'data-tab' ) );
+						this.changeActiveTab( event.currentTarget.getAttribute( 'data-tab' ), true );
```

Comment:
Move all `base-tabs` changes to the nested element

Reply:
Maybe we add it for tabs v1 also? there is no way we can do e2e test

Reply:
move to `base-tabs-v2`

Reply:
Handled.